### PR TITLE
Add major_incident_state to Flaw API and update filters

### PR DESCRIPTION
**Pull request by jobselko**
Related to "Improve Major Incident model and API" (OSIDB-266).

### CODE REVIEW

File: osidb/serializer.py [hunk 1]
```diff
@@ -796,6 +796,7 @@ class Meta:
                 "cvss3_score",
                 "nvd_cvss3",
                 "is_major_incident",
+                "major_incident_state",
```
Comment (jobselko) [inline osidb/serializer.py:799]: I am still unsure if this is all that needs to be done.

Comment (osoukup) [reply osidb/serializer.py:799]: Can you perform an API query filtering flaws with a specific `major_incident_state` value?

Comment (jobselko) [reply osidb/serializer.py:799]: Yes. If I do `http://localhost:8000/osidb/api/v1/flaws?major_incident_state=REJECTED`, I get `count: 2`.

**Review by osoukup — APPROVED** [review]
LGTM

**Pull request merged**